apps/site/src/app/(home)/sections/FAQ/FAQ.module.scss

```diff
@@ -1,121 +1,112 @@
 @use "zothacks-theme" as theme;
 @use "bootstrap-utils" as utils;
 
-.faq {
-	position: relative;
-	display: flex;
-	flex-direction: column;
-	width: 80%;
+$item-padding-y: 32px;
+$item-padding-x: 48px;
+
+$answer-font-size: 16px;
+$question-font-size: 20px;
```

Comment:
Thought: rather than using fixed font sizes, it would be better to set font sizes in rem units to account for different browser default font sizes